build/yamls/audit-webhook.yaml

```diff
@@ -0,0 +1,133 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: audit-pv-volume
+  labels:
+    type: local
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 5Gi 
+  accessModes:
+    - ReadWriteMany
+  hostPath:
+    path: "/home/vagrant/audit"
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: audit-pv-claim
+spec:
+  storageClassName: manual
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 4Gi
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: audit-deployment
+  labels:
+    app: audit
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: audit
+  template:
+    metadata:
+      labels:
+        app: audit
+    spec:
+      volumes:
+      - name: audit-pv-storage
+        persistentVolumeClaim:
+          claimName: audit-pv-claim
+      containers:
+      - name: audit
+        args: ["-d", "/data", "-v", "2"]
+        env:
+        - name: SERVICEACCOUNT_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.serviceAccountName
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        image: stanleywbwong/audit-webhook
+        ports:
+        - containerPort: 8080
+        volumeMounts:
+        - mountPath: "/data"
+          name: audit-pv-storage
+        securityContext:
+          runAsUser: 0000
+          runAsGroup: 0000
+      - name: audit-gui
+        image: stanleywbwong/audit-webui
+        volumeMounts:
+        - mountPath: "/data"
+          name: audit-pv-storage
+      serviceAccountName: audit-account
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: webhook-service
+spec:
+  selector:
+    app: audit
+  ports:
+    - name: webhook
+      protocol: TCP
+      port: 8080
+      targetPort: 8080
+  clusterIP: 10.96.0.100
```

Comment:
I would recommend not providing an IP address here as it may not match the ClusterIP CIDR for some users